src/shared/api/index.ts

```diff
@@ -1,19 +1,30 @@
+import type {
+  Account,
+  AssetDefinition,
+  Asset,
+  AssetSearchParams,
+  Block,
+  Domain,
+  Paginated,
+  PaginationParams,
+  TransactionSearchParams,
+  Transaction,
+  PeerStatus,
+} from '@/shared/api/dto';
+
 interface HTTPService {
   fetchAccounts: (params?: PaginationParams) => Promise<Paginated<Account>>
   fetchAccount: (id: string) => Promise<Account>
-  fetchAssets: (params?: PaginationParams) => Promise<Paginated<Asset>>
-  fetchAsset: (definition_id: string, account_id: string) => Promise<Asset>
-  fetchAssetDefinitions: (params: PaginationParams) => Promise<Paginated<AssetDefinition>>
+  fetchAssets: (params?: AssetSearchParams) => Promise<Paginated<Asset>>
+  fetchAsset: (id: string) => Promise<Asset>
```

Comment:
If you have a chance to use stronger types, please do!

```suggestion
  fetchAsset: (id: AssetId) => Promise<Asset>
```